src/components/App/index.spec.tsx

```diff
@@ -35,71 +40,125 @@ describe(__filename, () => {
       },
     };
 
-    const root = shallowUntilTarget(<App authToken={authToken} />, AppBase, {
+    const props = {
+      _log,
+      authToken,
+    };
+
+    return shallowUntilTarget(<App {...props} />, AppBase, {
       shallowOptions: { ...context },
     });
-
-    return root;
   };
 
-  it('renders without an authentication token', () => {
-    const root = render({ authToken: null });
+  it('renders without an authentication token', async () => {
+    const root = await render({ authToken: null });
```

Comment:
Why do you need `await` here? In the component itself, `componentDidMount` is not async, only `componentDidUpdate`.

Reply:
ahm, correct. This is not needed 😞